crates/bevy_core_pipeline/src/core_3d/mod.rs

```diff
@@ -366,6 +381,21 @@ impl PhaseItem for Transmissive3d {
     }
 }
 
+impl SortedPhaseItem for Transmissive3d {
+    // NOTE: Values increase towards the camera. Back-to-front ordering for transmissive means we need an ascending sort.
```

Comment:
This reminds me that distance is a misnomer if an ascending sort is back to front ordering as distance isn’t signed and that makes this difficult to read. Nothing new though, just noting it.